assets/wire/components/FollowTopicModal.jsx

```diff
@@ -41,23 +49,37 @@ class FollowTopicModal extends React.Component {
                 <h1>{gettext('Follow topic')}</h1>
                 <form onSubmit={this.onSubmit}>
                     <div className="form-group">
-                        <label htmlFor="topicName">{gettext('Topic name')}</label>
+                        <label htmlFor="topicName">{gettext('Name')}</label>
                         <input type="text"
                             className="form-control"
                             id="topicName"
-                            required="required"
+                            required={true}
                             value={this.state.label}
                             onChange={this.onChangeHandler('label')}
+                            maxLength={TOPIC_NAME_MAXLENGTH}
                         />
                     </div>
                     <div className="form-group">
-                        <label htmlFor="description">{gettext('Description')}</label>
-                        <textarea className="form-control"
+                        <label htmlFor="description">{gettext('Query')}</label>
+                        <input type="text"
```

Comment:
Fyi, there's an text input component for forms:
https://github.com/superdesk/newsroom/blob/master/assets/components/TextInput.jsx

It is the same thing but has a bit more in validation.